### ISSUE TITLE
Python3.5: Partial branch marked as fully covered

### ISSUE THREAD

**Issue by nedbat** [coveragepy/coveragepy#422]
*Originally reported by* **Florian Bruhin (Bitbucket: [The-Compiler](https://bitbucket.org/The-Compiler), GitHub: [The-Compiler](https://github.com/The-Compiler))**

----------------------------------------

When having a `module.py`:

```python
def func2(arg):
    if arg:
        pass

def func():
    func2(True)

if __name__ == '__main__':  # pragma: no branch
    func()
```

and a `.coveragerc`:

```ini
[run]
branch=True
```

and running `rm -r .coverage htmlcov &&  coverage run module.py && coverage html` that correctly marks line 2 as partially covered with a Python 3.4 coverage, but marks it as fully covered with Python 3.5.

----------------------------------------
- Bitbucket: https://bitbucket.org/ned/coveragepy/issue/422

**Comment by nedbat** [comment]
This will be fixed when the ast-branch is merged.

**Comment by nedbat** [comment]
This is fixed in [44719bd (bb)](https://bitbucket.org/ned/coveragepy/commits/44719bd), which will be 4.1
